scripts/test-make-requirements.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 set -e
 
-pip install pip-tools>=3.8
+# This is a dirty test. It can leave some files in a different state
+# than they were when it started.
```

Comment:
Would running `git checkout requirements/ requirements-py3/` before `exit 1` in the else statement fix this?